src/index.js

```diff
@@ -1,33 +1,76 @@
 import './style.css';
 
-const list = [
-  {
-    index: 0,
-    description: 'Complete Tommorows project',
-    completed: false,
-  },
-  {
-    index: 1,
-    description: 'Study',
-    completed: false,
-  },
-  {
-    index: 2,
-    description: 'Wash the dishes',
-    completed: true,
-  },
-];
-
-const toDoList = () => {
-  const List = document.getElementById('list');
-  for (let i = 0; i < list.length; i += 1) {
-    const li = document.createElement('li');
-    li.classList = 'list-item';
-    li.innerHTML = `
-      <input type='checkbox'>
-      <h3>${list[i].description}</h3>
-      `;
-    List.appendChild(li);
+const inputBox = document.getElementById('list-input');
+const addBtn = document.getElementById('submit-new-item');
+const deleteAllBtn = document.getElementById('clear-btn');
+const todoList = document.getElementById('list');
+
+class Task {
+  constructor(index, description, completed = false) {
+    this.index = index;
+    this.description = description;
+    this.completed = completed;
+  }
+}
+
+let listArray = [];
+
+addBtn.addEventListener('click', () => {
+  const userEnteredValue = inputBox.value;
+  const getLocalStorageData = localStorage.getItem('List item');
+  if (getLocalStorageData == null) {
+    listArray = [];
+  } else {
+    listArray = JSON.parse(getLocalStorageData);
   }
+  listArray.push(userEnteredValue);
+  localStorage.setItem('List item', JSON.stringify(listArray));
+  addBtn.classList.remove('active');
+});
+
+const showTasks = () => {
+  const getLocalStorageData = localStorage.getItem('List item');
+  if (getLocalStorageData == null) {
+    listArray = [];
+  } else {
+    listArray = JSON.parse(getLocalStorageData);
+  }
+  let newLiTag = '';
+  listArray.forEach((element) => {
+    newLiTag += `
+            <li class = 'list-item'>
+            <input id= '${Task.index}' class='checkbox' type='checkbox' ${!Task.completed ? '' : 'checked'}>
+            ${element}<span class="icon"><i class="fas fa-trash"></i></span>
+            </li>
+            `;
+  });
```

Comment:
- [ ] Kindly implement the edit functionality which is part of the project requirement

- [ ] Let's implement the task completed functionality. Currently, there is no record to show if a task is completed or not. By default, this attribute should be set to false. Kindly see the image below.

![todolist](https://user-images.githubusercontent.com/53124548/184361662-10e71197-4add-4999-8760-1ce4b2046818.png)